torch_geometric/nn/functional/spmm.py

```diff
@@ -0,0 +1,22 @@
+import torch
+from torch import Tensor
+from torch_sparse import SparseTensor
+
+from torch_geometric.utils import is_torch_sparse_tensor
+
+
+def spmm(src: SparseTensor, other: Tensor, reduce: str = "sum") -> Tensor:
```

Comment:
I personally think we should move this to `utils` as well since I see it potentially used in other places as well, e.g., transforms.

Reply:
Ok, got it.